recipes/pycoalescence/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pycoalescence" %}
+{% set version = "1.2.6rc49" %}
+{% set sha256 = "b738f2930883e073cd66fbc2c033f60235d10f68f28716e97018787324115d07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"  # [win]
+  detect_binary_files_with_prefix: True
+  skip: True  # [(win and vc<14) or linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake >=3.8
+
+  host:
+    - python
+    - pip
+    - setuptools
+    - boost
+    - gdal >=2.1.0
+    - sqlite
+    - matplotlib
+    - numpy
+
+  run:
+    - python
+    - boost
+    - gdal >=2.1.0
+    - sqlite
+    - matplotlib
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pycoalescence
+    - pycoalescence.necsim
+
+about:
+  home: https://bitbucket.org/thompsonsed/pycoalescence/
+  license: MIT
+  license_file: LICENSE.txt
+  summary: 'Spatially-explicit neutral ecology simulator using coalescence methods'
+  description: |
+    pycoalescence is an API for easy interfacing with the necsim c++ neutral
+    simulator for running ecological models under the effects of
+    dispersal, speciation and ecological drift in a spatially-explicit
+    manner.
+  doc_url: http://http://pycoalescence.readthedocs.io/
```

Comment:
Argh, just realized this URL is broken. @thompsonsed can you fix that in the feedstock?

Reply:
Yep, my bad, will do once it's up